.jenkins/Jenkinsfile

```diff
@@ -150,8 +145,13 @@ def win2016LinuxElfBuild(String build_type) {
             cleanWs()
             checkout scm
             unstash "linux${build_type}"
-            PowerShellWrapper('mv build linuxbin')
-            PowerShellWrapper("./scripts/test-build-config.ps1 -add_windows_enclave_tests -build_type ${build_type} -linux_bin_dir ${WORKSPACE}/linuxbin/tests")
+            bat 'move build linuxbin'
+            dir('build') {
+                bat """vcvars64.bat x64 && \
+                       cmake.exe -G \"Visual Studio 15 2017 Win64\" -DADD_WINDOWS_ENCLAVE_TESTS=1 -DCMAKE_BUILD_TYPE=${build_type} -DLINUX_BIN_DIR=${WORKSPACE}\\linuxbin\\tests ${WORKSPACE} && \
```

Comment:
`${WORKSAPCE}` is a Jenkins Pipeline variable that points to the root of the git project. In this case it would look something like this: `D:\Jenkins\workspace\$job_name\openenclave`.

Reply:
ok, that should work.